src/term/mod.rs

```diff
@@ -1734,7 +1701,6 @@ impl From<Term> for RichTerm {
 impl std::fmt::Display for RichTerm {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         use crate::pretty::*;
-        use pretty::BoxAllocator;
```

Comment:
Any reason to move this specific `use` from the local function to the top-level? We don't really have consistent rules right now about where to place `use` statements, but I vaguely intuit that it makes more sense to keep it local if it's very specific to the current function, and doesn't have much to do with the rest of the module, which seems to be the case here.

Reply:
That's a very good question. It might have happened during a previous iteration, but right now there's no good reason. I think I'll revert this part.